### PR TITLE
feat(vantiv): add ThreatMetric tags insertion

### PR DESCRIPTION
**Pull request by Jisay**
Global refactor is on the way with the next payment method registration implementation.

### CODE REVIEW

File: src/components/register/iframeVantiv/directive.js [hunk 1]
```diff
@@ -17,17 +21,82 @@ export default {
...
+    /**
+     *  Insert a script with given attributes
+     */
+    const insertElement = (tagName, attributes = {}, events = {}, styles = {}, options = {}) => {
+      const element = document.createElement(tagName);
+
+      // set attributes of dom element
+      Object.keys(attributes).forEach((key) => {
+        element.setAttribute(key, get(attributes, key));
+      });
```
Comment (marie-j) [inline src/components/register/iframeVantiv/directive.js]: Maybe a `for attribute in attributes` could be used here

**Comment by jleveugle** [comment]
Hello @Jisay,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: src/components/register/iframeVantiv/directive.js (hunk 1 above)
Comment (Jisay) [reply src/components/register/iframeVantiv/directive.js]: What's the gain of using `for attribute in attributes`?

File: src/components/register/iframeVantiv/directive.js [hunk 2]
```diff
@@ -17,17 +21,82 @@ export default {
...
+      }
+
+      return element;
+    };
+
+    // declare insertThreatMetric method that will load the script and iframe
+    // that will handle Vantiv ThreatMetric mechanism
+    vantivIframeCtrl.insertThreatMetric = ({
+      formSessionId,
+      organizationId,
```
Comment (blaryjp) [inline src/components/register/iframeVantiv/directive.js]: where is the variable defined?

File: src/components/register/iframeVantiv/constants.js [hunk 3]
```diff
@@ -1,3 +1,15 @@
+export const THREAT_METRIC = {
+  PAGE_ID: 1,
+  SCRIPT: {
+    id: 'threat_metric_script',
+    src: 'https://h.online-metrix.net/fp/tags.js',
+  },
+  IFRAME: {
+    id: 'threat_metric_iframe',
+    src: ' https://us.ovhcloud.com/order/threatmetrix',
```
Comment (blaryjp) [inline src/components/register/iframeVantiv/constants.js]: maybe we can increase the security by adding an Integrity attr?

**Pull request merged**